src/uu/tail/src/args.rs

```diff
@@ -182,19 +182,53 @@ impl Settings {
     }
 
     pub fn from(matches: &clap::ArgMatches) -> UResult<Self> {
-        let mut settings: Self = Self {
-            follow: if matches.get_flag(options::FOLLOW_RETRY) {
-                Some(FollowMode::Name)
-            } else if matches.value_source(options::FOLLOW) != Some(ValueSource::CommandLine) {
-                None
-            } else if matches.get_one::<String>(options::FOLLOW)
-                == Some(String::from("name")).as_ref()
+        // We're parsing --follow, -F and --retry in one go under the following conditions:
+        // * -F sets --retry and --follow=name (the -F flag is restricted by clap to be set only
+        // once in contrast to gnu's tail which allows this flag to be specified multiple times)
+        // * plain --follow or short -f is the same like specifying --follow=descriptor and can
+        // overwrite itself (specified multiple times)
+        // * specifying --retry together with -F doesn't change the setting of `retry` to true. The
+        // --retry flag itself is restricted by clap to be specified only once (unlike gnu's tail)
```

Comment:
Can't we match GNU by allowing retry multiple times?

Reply:
Sure. That's just the way it was before this pr and I wasn't sure. I think `-F` should be allowed also multiple times

Reply:
To be fully compliant with gnu's tail, I think we should set `clap_builder::builder::Command::args_override_self` to true globally for all arguments and not only for specific ones. Afaik, this is the way how they parse arguments. I could do this in another pr, maybe together with unit testing the arguments.